ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -0,0 +1,43 @@
+import Route from '@ember/routing/route';
+
+export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
+  // =methods
+
+  /**
+   * Load a target.
+   * @param {object} params
+   * @param {string} params.target_id
+   * @return {TargetModel}
+   */
+  async model({ target_id }) {
+    return this.store.findRecord('target', target_id, { reload: true });
+  }
+
+  /**
+   * Navigate to hostsets within a target.
+   * @param {TargetModel} model 
+   */
+  afterModel(model) {
+    this.transitionTo('scopes.scope.projects.targets.target.hosts', model);
+  }
```

Comment:
Is there a better way to load into hosts route? I could have used 'index' route and places hosts templates in it but we called this out as a separate route on admin.

Reply:
The redirect should happen in the target/index route, using the `redirect()` route method.  You also do not need to (and shouldn't) pass the model.  The target/hosts route does not take a target instance.